code_review segfaults

lack of check for nil in [code_review.go](https://github.com/ossf/scorecard/blob/main/checks/code_review.go#L110)
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x8f7b1c]
```

--- a/checks/code_review.go
+++ b/checks/code_review.go
@@ -107,7 +107,8 @@ func IsPrReviewRequired(c *checker.CheckRequest) checker.CheckResult {
 	if err != nil {
 		return checker.MakeInconclusiveResult(codeReviewStr)
 	}
-	if bp.GetRequiredPullRequestReviews().RequiredApprovingReviewCount >= 1 {
+	if bp.GetRequiredPullRequestReviews() != nil &&
+		bp.GetRequiredPullRequestReviews().RequiredApprovingReviewCount >= 1 {
 		c.Logf("pr review policy enforced")
 		const confidence = 5
 		return checker.CheckResult{

